cmfpy/datasets/vox_celeb.py

```diff
@@ -0,0 +1,104 @@
+import numpy as np
+import glob, os
+import scipy.io.wavfile as wavfile
+import scipy.signal as signal
+import sklearn.preprocessing as preprocessing
+import warnings
+
+DATAPATH = os.path.expanduser("~/cmf_data/VoxCeleb/")
+
+class VoxCeleb:
+    """Spectrogram data from the Vox Celeb Dataset."""
+
+    def __init__(self, secs, path, concat=True):
```

Comment:
Could we possibly also include the starting time? So someone could load the last five minutes for example.

Reply:
We could, it gets a little tricky though because I think in most cases the user will be passing a path to a folder that contains multiple files – so in that case the question becomes: the last five seconds of which file?